ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -47,6 +48,33 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
   func application(_ application: UIApplication,
                    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
+
+    // First things first:  enable Sentry for crash reporting.
+    do {
+      Sentry.Client.shared = try Sentry.Client(dsn: "https://d14d2efb594e4345b8367dbb61ebceaf@sentry.keyman.com/8")
```

Comment:
For iOS, that _is_ the global constant.  It's forced on us by the Cocoa implementation of Sentry.  Or are we talking the `dsn` code?  If that code, note that the numeral at the end of the URL changes per project.

Reply:
Yes, I'm talking about just the DSN constant. If possible, it should be in a separate file because it's just cleaner for future maintenance (global search and replace is of course not difficult but it's easy to forget a platform if the dsn is hidden away).

Reply:
I'm... not immediately clear on how to make it a constant (like with VERSION.md and TIER.md) that can be imported as a value to Swift code.  That might take me a bit; would it be fine to proceed with the current state and make that common DSN abstraction an issue?

Reply:
Yep. It's low priority for sure.

Reply:
Documented as issue #2792.